apps/src/gamelab/GameLab.js

```diff
@@ -48,9 +48,16 @@ import {
 } from '../containedLevels';
 import { hasValidContainedLevelResult } from '../code-studio/levels/codeStudioLevels';
 import {actions as jsDebugger} from '../lib/tools/jsdebugger/redux';
+import project from '@cdo/apps/code-studio/initApp/project';
```

Comment:
Nit: I've been trying to prefer relative paths to for imports, because (at least for me) refactoring tools handle them better.